src/libraries/System.Text.Json/gen/JsonSourceGenerator.Parser.cs

```diff
@@ -75,6 +75,7 @@ private sealed class Parser
 
             private readonly Type _booleanType;
             private readonly Type _charType;
+            private readonly Type _timeSpanType;
```

Comment:
nit: this should be nullable since the type is not included in the `SpecialType` enum.